pkg/controller/controller.go

```diff
@@ -328,6 +328,13 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 		panic(err)
 	}
 
+	// If not defined, default to 6
+	if cont.config.AciServiceMonitorInterval == 0 {
+		cont.config.AciServiceMonitorInterval = 6
+	}
+	cont.log.Info("AciServiceMonitorIntervalPodIpPoolChunkSize conf is set to: ", cont.config.AciServiceMonitorInterval)
+
+
```

Comment:
This is added mainly for testing; I will remove before merge.

Reply:
Please set to Debug, instead of removing it.

Reply:
I meant to say - I will remove this code of setting IPSLA value to positive number with-in ACC since this is exposed from acc-provision and we need to honor the setting coming from user.